### DISCUSSION TITLE
Kindly add your references

### DISCUSSION THREAD

**Discussion by Akshat111111** [Akshat111111/Hedging-of-Financial-Derivatives#104] [Ideas]
@misskiran @apooyadv @tejasvinigoel @Divyanshi1002 @Pranshu-jais @diptarup794 @bhartijoshi04 @amishhaa @Amarta113 @aryan1165 @alishasingh06 Kindly put your references over here ie the the content you have referred in order to learn more about financial programming. Then I will put it in the readme so that newcomers who not have any knowledge can start from scratch.Kindly put minimum 3 references.

**Comment by misskiran** [comment]
My References for NSE-Arima Implementation were :
1.https://www.analyticsvidhya.com/blog/2020/10/how-to-create-an-arima-model-for-time-series-forecasting-in-python/
2.https://www.youtube.com/watch?v=2XGSIlgUBDI&t=325s
3.https://medium.com/analytics-vidhya/a-thorough-introduction-to-arima-models-987a24e9ff71
4.https://www.youtube.com/watch?v=8FCDpFhd1zk
5.https://www.kaggle.com/code/nageshsingh/stock-market-forecasting-arima

**Comment by bhartijoshi04** [comment]
The references I used while analyzing NIFTY-50 data from May 2009- May 2024
1. https://finance.yahoo.com/quote/%5ENSEI/history  [Downloaded the historical data from Yahoo Finance]
2.  https://thepythonlab.medium.com/mastering-financial-data-analysis-with-python-ffd31468f964#:~:text=Python%20has%20emerged%20as%20a,versatility%2C%20and%20extensive%20library%20support. [ read article from medium to get more familiarity with the topic]
3. https://plotly.com/python  [For understanding plots, I used in the data analysis]

**Comment by Divyanshi1002** [comment]
The references for KNN model.
1) Historical dataset 10-05-2024 to 10-05-2023 (https://finance.yahoo.com/quote/%5ENSEI/history)
2)To understand more about KNN (https://medium.com/analytics-vidhya/day-two-in-machine-learning-cd5b5853c18b)
3)To know about [sklearn.neighbors](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.neighbors).KNeighborsClassifier (https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html)

**Comment by Pranshu-jais** [comment]
The references I used  for  Real Estate Price Prediction Model
1) https://www.kaggle.com/amitabhajoy/bengaluru-house-price-data  [ downloaded bengaluru_house_price data from kaggle]
2) https://medium.com/@yash.shejwal20/house-price-prediction-system-using-ml-algorithms-e7ef83ded562 [[ read article from medium to get more familiarity with the topic]
3) To understand more about linear regression https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html
4)  T o understand about different  Predictive Modeling  https://www.geeksforgeeks.org/what-is-predictive-modeling/
